Makefile

```
@@ -205,12 +190,12 @@ lint-spellcheck:
 lint-yaml: ## Lint yaml files
 	yamllint .
 
-.PHONY: fmt-md ## Format markdown files
-fmt-md:
+.PHONY: fmt-md
+fmt-md: ## Format markdown files
 	npx --yes prettier --write --prose-wrap always **/*.md
```

I had made a mistake here and I forgot the quotes 🤦 Maybe we can fix it at the same time?

```suggestion
	npx --yes prettier --write --prose-wrap always "**/*.md"
```